libraries/heft-config-file/package.json

```diff
@@ -31,6 +31,7 @@
     "@rushstack/heft": "0.68.12",
     "@rushstack/heft-node-rig": "2.6.48",
     "@types/heft-jest": "1.0.1",
-    "@types/node": "20.17.19"
+    "@types/node": "20.17.19",
```

Comment:
Now that we're using Node 20 types, should the min supported Node version in the rush.json be updated?

Reply:
Node 18 is technically still supported. We're still testing with it.